cmd/utils/nodecmd/defaultcmd.go

```diff
@@ -162,7 +163,52 @@ func CheckCommands(ctx *cli.Context) error {
 	return nil
 }
 
-func BeforeRunKlaytn(ctx *cli.Context) error {
+func contains(list []cli.Flag, item cli.Flag) bool {
+	for _, flag := range list {
+		if flag.GetName() == item.GetName() {
+			return true
+		}
+	}
+	return false
+}
+
+func union(list1, list2 []cli.Flag) []cli.Flag {
+	for _, item := range list2 {
+		if !contains(list1, item) {
+			list1 = append(list1, item)
+		}
+	}
+	return list1
+}
+
+func allNodeFlags() []cli.Flag {
+	nodeFlags := []cli.Flag{}
+	nodeFlags = append(nodeFlags, CommonNodeFlags...)
+	nodeFlags = append(nodeFlags, CommonRPCFlags...)
+	nodeFlags = append(nodeFlags, ConsoleFlags...)
+	nodeFlags = append(nodeFlags, debug.Flags...)
+	nodeFlags = union(nodeFlags, KCNFlags)
+	nodeFlags = union(nodeFlags, KPNFlags)
+	nodeFlags = union(nodeFlags, KENFlags)
+	nodeFlags = union(nodeFlags, KSCNFlags)
+	nodeFlags = union(nodeFlags, KSPNFlags)
+	nodeFlags = union(nodeFlags, KSENFlags)
+	return nodeFlags
+}
+
+func FlagsFromYaml(ctx *cli.Context) error {
+	if ctx.String("conf") != "" {
```

Comment:
What about adding a global variable like this instead of using the hard-coded string.

`var confFile = "conf"`

Reply:
Good, Thanks